src/main/preload/mattermost.js

```diff
@@ -373,3 +374,25 @@ ipcRenderer.on(CALLS_JOIN_REQUEST, (event, message) => {
 window.addEventListener('resize', () => {
     ipcRenderer.send(VIEW_FINISHED_RESIZING);
 });
+
+let isPasswordBox = false;
+
+window.addEventListener('focusin', (event) => {
+    if (isPasswordBox) {
+        return;
+    }
+
+    if (event.target.tagName === 'INPUT' && event.target.type === 'password') {
+        ipcRenderer.send(TOGGLE_SECURE_INPUT, true);
+        isPasswordBox = true;
+    }
+});
+
+window.addEventListener('focusout', () => {
```

Comment:
This might not work if you switch between two password input boxes like we probably have for confirming your password in the settings modal.

Instead of clearing `isPasswordBox` on `focusout`, what if we use a single focus handler and write it something like
```js
// obviously, this needs a better name
const isNowPasswordBox = event.target.tagName === 'INPUT' && event.target.type === 'password'; 
if (isNowPasswordBox && !isPasswordBox) {
    // turn on secure
} else if (!isNowPasswordBox && isPasswordBox) {
    // turn off secure
}

isPasswordBox = isNowPasswordBox;
```
That way, it'll stay active when you move between password inputs, and it'll only need one handler so you don't need to worry about event order either